kuma/wiki/tests/test_ssr.py

```diff
@@ -3,22 +3,25 @@
 
 import mock
 import pytest
+import requests.exceptions
+from django.conf import settings
```

Comment:
Just FYI, you could use the `settings` fixture provided by `pytest-django` (https://pytest-django.readthedocs.io/en/latest/helpers.html#settings) in your tests below, but since you're not modifying any settings, it's fine to use this.

Reply:
I've changed it to use the fixture. I'm happy to learn about that!